low-level/solution/lr2/server/manifest.py

```diff
@@ -336,15 +340,26 @@ def get_os_server_info(self):
         if specifics:
             os_info = {
                 "uid": specifics.get("id", "NA"),
-                "type": "os",
+                "type": "software",
                 "description": "OS information",
                 "product": specifics.get("pretty_name", "NA"),
-                "manufacturer": specifics.get("manufacturer_name", "NA"),
-                "serial_number": "NA",
-                "part_number": "NA",
+                "manufacturer": specifics.get("manufacturer_name",
+                                    "Not Applicable"),
+                "serial_number": "Not Applicable",
+                "part_number": "Not Applicable",
                 "version": specifics.get("version", "NA"),
                 "last_updated": int(time.time()),
                 "specifics": [specifics]
             }
             os_data.append(os_info)
         return os_data
+
+    @staticmethod
+    def get_local_disk(disks):
```

Comment:
Again add a TODO here, that we need better logic to identify server local disks

Reply:
Done